assets/_sass/_search.scss

```diff
@@ -0,0 +1,36 @@
+// https://github.com/mauriciogentile/angular-livesearch/blob/1f4e357ab3b50701eeedcfb45c12cd901d612087/example/styles.css#L31-L65
+ul.searchresultspopup {
```

Comment:
Honestly, I copied-and-pasted this file from the URL above – didn't really spend any time cleaning it up. Worth a pass to delete anything we don't need, I gueeeeesssss :expressionless: